app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -119,6 +122,18 @@ def pvm_instances
         :read_only    => true
       )
 
+      # saving placement_group
+      placement_group = collector.placement_group(instance.placement_group)
+      value = "#{placement_group.name}[#{placement_group.policy}]" unless placement_group.nil?
```

Comment:
If `placement_group.nil?` does `value == nil`? or is it left unitialized?

Reply:
`value == nil`  will be true

Reply:
Great, thanks for the explanation... makes sense and I learned something new about Ruby :)

Reply:
If `placement_group` is `nil` do you want to build the advanced setting at all?  Seems confusing to have the advanced setting with a value of `nil`

What about
```ruby
placement_group = collector.placement_group(instance.placement_group)
if placement_group
    persister.vms_and_templates_advanced_settings.build(
      :resource     => ps_vmi,
      :name         => 'placement_group',
      :display_name => _('Placement Group'),
      :description  => _('The placement group of the server'),
      :value        => "#{placement_group.name}[#{placement_group.policy}]",
      :read_only    => true
    )
end
```

Reply:
@agrare I thought adding an Placement group = nil entry into the advanced settings was a good idea (to differentiate between not collecting placement group vs. no placement is assigned to a VM). I was also thinking about reporting, but I'm not completely sure what the behavior is - could we filter a report of VMs without placement groups assigned in the same way that we could with placement groups = nil?

Reply:
I just noticed that your other optional advanced setting didn't build if it was nil down on `+134` [[ref]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/266/files#diff-f985d1bd514a044797c4062d1d8d94ff3da33e97f6bfa8d182955661184e7d45R133)

Reply:
But if you'd prefer to create it with a nil value that works too

Reply:
If both ways work (sounds like they do) then I don't feel strongly either. @Kuldip-Nanda you can make the call if you'd like since it's your patch.

Reply:
Let it remain Adam. This is a powerful feature. If it is not set it will show the customer something he can use.